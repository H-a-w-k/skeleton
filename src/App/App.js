import React from "react";
import "./App.scss";
import AppInput from "./AppInput";

const App = () => {
  return (
    <div className="main">
      <p>Hello my world from App</p>
      <AppInput></AppInput>
    </div>
  );
};

export default App;
