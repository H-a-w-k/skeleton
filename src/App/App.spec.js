import React from "react";
import App from "./App";

describe("App", () => {
  it("snapshot", () => {
    jestExpect(shallow(<App />).debug()).toMatchSnapshot();
  });

  it("should render", () => {
    expect(shallow(<App />).exists()).true;
  });
});
