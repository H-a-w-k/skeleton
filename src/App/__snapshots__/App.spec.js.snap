// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot 1`] = `
"<div className=\\"main\\">
  <p>
    Hello my world from App
  </p>
  <AppInput />
</div>"
`;
